[2024-11-05 17:58:58,566][root][INFO] - ---------------------------------------------------------------
[2024-11-05 17:58:58,567][root][INFO] - Env:
CUDA_VISIBLE_DEVICES: null
PYTHONOPTIMIZE: null

[2024-11-05 17:58:58,568][root][INFO] - Command line args:

[2024-11-05 17:58:58,568][root][INFO] - ---------------------------------------------------------------
[2024-11-05 17:58:58,570][root][INFO] - Run config:
device: cuda:4
solver_fn: nk.optimizer.solver.cholesky
lr: 0.0001
diag_shift: 1.0e-10
n_iter: 2000
chunk_size_jac: 2
chunk_size_vmap: 1
save_every: 3
base_dir: /scratch/.amisery/grad_sample/
model:
  _target_: grad_sample.model.ising.TFI
  L: 3
  h: 1.0
ansatz:
  _target_: nk.models.RBM
  alpha: 1
task:
  _target_: grad_sample.tasks.fullsum_train.Trainer

[2024-11-05 17:58:58,571][root][INFO] - ---------------------------------------------------------------
[2024-11-05 17:59:01,045][root][CRITICAL] - Error in call to target 'grad_sample.tasks.fullsum_train.Trainer':
TypeError("descriptor '__init__' requires a 'super' object but received a 'DictConfig'")
full_key: task
Traceback (most recent call last):
  File "/scratch/.amisery/netket2/lib/python3.10/site-packages/hydra/_internal/instantiate/_instantiate2.py", line 92, in _call_target
    return _target_(*args, **kwargs)
  File "/home/amisery/phd/tfi_gs_samples/grad_sample/src/grad_sample/tasks/fullsum_train.py", line 16, in __init__
    super.__init__(cfg)
TypeError: descriptor '__init__' requires a 'super' object but received a 'DictConfig'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/amisery/phd/tfi_gs_samples/grad_sample/src/grad_sample/main.py", line 49, in main
    task = instantiate(cfg.task, cfg)
  File "/scratch/.amisery/netket2/lib/python3.10/site-packages/hydra/_internal/instantiate/_instantiate2.py", line 226, in instantiate
    return instantiate_node(
  File "/scratch/.amisery/netket2/lib/python3.10/site-packages/hydra/_internal/instantiate/_instantiate2.py", line 347, in instantiate_node
    return _call_target(_target_, partial, args, kwargs, full_key)
  File "/scratch/.amisery/netket2/lib/python3.10/site-packages/hydra/_internal/instantiate/_instantiate2.py", line 97, in _call_target
    raise InstantiationException(msg) from e
hydra.errors.InstantiationException: Error in call to target 'grad_sample.tasks.fullsum_train.Trainer':
TypeError("descriptor '__init__' requires a 'super' object but received a 'DictConfig'")
full_key: task
